x/group/server/invariants_test.go

```diff
@@ -339,3 +341,279 @@ func TestGroupTotalWeightInvariant(t *testing.T) {
 		require.Equal(t, spec.expErr, broken)
 	}
 }
+
+func TestTallyVotesSumInvariant(t *testing.T) {
+	curCtx, cdc, key := getCtxCodecKey(t)
+
+	// Group Account Table
+	groupAccountTableBuilder := orm.NewPrimaryKeyTableBuilder(GroupAccountTablePrefix, key, &group.GroupAccountInfo{}, orm.Max255DynamicLengthIndexKeyCodec{}, cdc)
+	groupAccountTable := groupAccountTableBuilder.Build()
+
+	// Group Member Table
+	groupMemberTableBuilder := orm.NewPrimaryKeyTableBuilder(GroupMemberTablePrefix, key, &group.GroupMember{}, orm.Max255DynamicLengthIndexKeyCodec{}, cdc)
+	groupMemberByGroupIndex := orm.NewUInt64Index(groupMemberTableBuilder, GroupMemberByGroupIndexPrefix, func(val interface{}) ([]uint64, error) {
+		group := val.(*group.GroupMember).GroupId
+		return []uint64{group}, nil
+	})
+	groupMemberTable := groupMemberTableBuilder.Build()
+
+	// Proposal Table
+	proposalTableBuilder := orm.NewAutoUInt64TableBuilder(ProposalTablePrefix, ProposalTableSeqPrefix, key, &group.Proposal{}, cdc)
+	proposalTable := proposalTableBuilder.Build()
+
+	// Vote Table
+	voteTableBuilder := orm.NewPrimaryKeyTableBuilder(VoteTablePrefix, key, &group.Vote{}, orm.Max255DynamicLengthIndexKeyCodec{}, cdc)
+	voteTable := voteTableBuilder.Build()
+
+	_, _, adminAddr := testdata.KeyTestPubAddr()
+	_, _, addr1 := testdata.KeyTestPubAddr()
+	_, _, addr2 := testdata.KeyTestPubAddr()
+
+	curBlockTime, err := gogotypes.TimestampProto(curCtx.BlockTime())
+	require.NoError(t, err)
+
+	specs := map[string]struct {
+		groupAccReq []*group.GroupAccountInfo
+		policy      group.DecisionPolicy
+		membersReq  []*group.GroupMember
+		proposalReq []*group.Proposal
+		voteReq     []*group.Vote
```

Comment:
suffix Req is not necessary. wdyt?

Reply:
yeah I also think that just having `groupAccs`, `members`, `proposals` and `votes` would be better

Reply:
Okay, I'll fix this for all invariants